web-client/public/js/grnstate.js

```diff
@@ -0,0 +1,3 @@
+export var grnState = {
+    dashedLine : $("#dashedGrayLineButton").prop("checked"),
```

Comment:
Remember that we said that `grnState` is _pure model_. This should just be `true` or `false`.